### ISSUE TITLE
ip_link.show(...) crashes for certain ip link layout

### ISSUE THREAD

**Issue by PeHobt** [diosney/node-iproute#9]
Hi, using iproute 1.0.2 I have the following trouble

Linux **ip link** layout is

```
 ip link
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: ens5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9001 qdisc aipt2 state UP mode DEFAULT group default qlen 1000
    link/ether 0a:ca:90:10:38:8e brd ff:ff:ff:ff:ff:ff
    altname enp0s5
    ...
```

i.e. the second link has 3 lines and it causes the following error

```
TypeError: Cannot read property 'slice' of undefined
at exports.parse_links (/opt/.../node_modules/iproute/lib/link/utils.js:95:35)
at /opt/.../node_modules/iproute/lib/link/show.js:65:21
at ChildProcess.exithandler (child_process.js:299:7)
at ChildProcess.emit (events.js:314:20)
at maybeClose (internal/child_process.js:1022:16)
at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
```
Would it be possible to adapt code to work correctly also with this **ip link** layout?

**Comment by diosney** [comment]
I just published version `2.0.0-beta`, which is an overhaul of the library and fixes a lot of issues, this one included, since it uses now the native iproute `-json` option to get parsed results out of the box in `show` methods, so you can consider it as fixed in `2.0`.

The beta tag is mainly to keep it a week or two for you to test it and let me know of anything before publishing the final version, but it should be ok.

**Comment by diosney** [comment]
You can test the beta by issuing `npm install iproute@beta`.
